ocaml/xapi-cli-server/record_util.ml

```diff
@@ -706,6 +706,24 @@ let host_sched_gran_to_string = function
   | `socket ->
       "socket"
 
+let host_numa_affinity_policy_to_string = function
+  | `any ->
+      "any"
+  | `besteffort ->
+      "best-effort"
+  | `default ->
+      "default"
+
+let host_numa_affinity_policy_of_string = function
+  | "any" ->
+      `any
+  | "besteffort" ->
+      `besteffort
+  | "default" ->
+      `default
+  | _ ->
+      failwith "Bad type"
```

Comment:
should we use the other examples around and raise a Record_failure instead with the expected valid values?

Reply:
I did, but followed the wrong example that used failwith. Updated (btw this file should really be  mostly autogenerated).